docs/configuration-reference/components/linkerd.md

```diff
@@ -0,0 +1,58 @@
+# Linkerd configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Attribute reference](#attribute-reference)
+* [Applying](#applying)
+* [Deleting](#deleting)
```

Comment:
Format should be changed.

Reply:
I am not sure if I follow about what needs to change?

Reply:
Ah gotcha. NVM